src/orga/guidelines.rst

```diff
@@ -164,3 +164,21 @@ The guidance that we can provide here is two fold:
    For example, we're holding off on turning off py36 until pypy comes out with pypy3.7.
 3. If there are lots of people in the community relying on older versions, core team can decide to keep an old version around
    For example, we held off turning off py27 even after numpy, scipy dropped support as there were many in the community interested in keeping support until the end of life of that version.
+
+
+Requesting ``core`` to Merge PRs
+================================
+
+Due to various reasons, it is sometimes necessary for the ``core`` team to merge PRs on feedstocks they do not personally
+maintain. The ``core`` team abides by the following guidelines when doing this.
+
+1. The ``core`` team will normally wait three days before merging a PR on a feedstock they do not maintain personally. This
+   rule encompasses both migration PRs from the autotick bot and requests from community members to merge PRs on feedstocks.
+2. In extenuating circumstances (i.e., bad bugs, blocking lots of other PRs/work), the ``core`` team reserves the right to merge
+   before the three day window has passed. In this case, at least two ``core`` members must approve the merge. Approvals include
+   the act of merging and a ``core`` member making the PR being merged.
+3. The ``core`` team will do its best to not merge PRs marked as in a draft state, either explicitly as a github draft PR or
+   through a label, comment, or indication in a title.
+4. The ``core`` team will do its best to not merge PRs with requested changes (via a comment, label, title, or a PR review),
+   though if the community member requesting the change is unresponsive to questions or to changes being made,
+   then they may merge.
```

Comment:
I'm wondering if a point should be added that PRs might be merged immediately if there are other PRs which have not been interacted with. I'm thinking about a case like:

1. The bot has made a couple of update PRs which were passing at the time
2. Two months pass with no other maintainer interactions in the repository
3. Someone comes along, rebases all of the PRs into a new PR and asks for it to be merged

How about adding:

```markdown
5. If there are other pull requests in the repository for which the feedstock maintenance team
   have not interacted, this may be taken as an indication the maintenance team is inactive
   causing new pull requests to be merged more promptly. Activity on other pull requests or issues
   in the repository will always invalidate this point.
```

Reply:
This should require a second member of core. I will add it to the example list of exceptions.

Reply:
See my latest commit.